src/components/Database/Database.tsx

```diff
@@ -56,7 +56,7 @@ export const Database: React.FC<Props> = ({ config, namespace }) => {
   );
 
   return (
-    <div className="Database">
+    <div className="Database-Database">
       <ThemeProvider options={theme}>
```

Comment:
Do we need a themeProvider here if there is one higher-up? Is anything changing from the global theme?

Reply:
Yes, primary color (like modal button) is different. Whether we want that or not is a different question though, but let's keep it like that until we do the polishing rounds later.